html/ghShared.py

```diff
@@ -28,6 +28,7 @@
 import math
 
 BASE_SCRIPT_URL = '/'
+BASE_WEB_DOMAIN = 'https://www.galaxyharvester.net' # Update to the root domain name for your site. Used for email verification link do not include trailing /
```

Comment:
I’m not sure how the test site is deployed. Is that outlined somewhere? I could be wrong, but I assume that _this_ file isn’t manipulated in the process (and might require more code to handle that automation). 

Did you consider using a environment variable, and maybe defaulting to the default domain if not present?

But again, my assumption may be off base. 🤷‍♂️

Reply:
This file is essentially the shared config file and is called as an import in createuser.py anyway.

If someone were to deploy this to another domain, they would need to add the domain regardless, so an if true would just be needless extra steps

Reply:
You are right, however, registering environment variables is best practice for config items like this, same would be true of the RECAPTCHA_SITEID also present in this file. 

My assumption of the test site is that it is essentially just the same as running the site on local machine

Reply:
I hear ya. Yea, I don’t really know _how_ the app is deployed in each environment (and opinion doesn’t mean much in this scenario), so I’ll let Paul take it from here 😁

Reply:
Although I will admit that the test URL may be present in the configured recaptcha URLs for the main site, which I am only mentally processing now... it's 3:35am.

Reply:
Yeah, I haven't built any fancy deployment systems for the 2 sites so that the right values are translated here, essentially there are just a couple files like this one that have to be manually tweaked after deployment if the target is the test site.  Probably just separating that data from the code files is best as already stated.  A far as how things work now though, this change creates a workable data point to easily customize for each site instance.

Reply:
Gotcha. Yea, Python definitely isn't my go-to scripting language, but I think env vars could work with something like:

```python
import os

# other code

BASE_WEB_DOMAIN = os.environ.get('BASE_WEB_DOMAIN') or 'https://www.galaxyharvester.net'
```

If you have a simple way to set environment variables in your deploy process, it may be worth considering. I'll leave it at that. 👍